jekyll-seo-tag.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "jekyll", "~> 3.3"
-  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_development_dependency "bundler", "~> 2.0.1"
```

Comment:
We don't want to force bundler 2 for now, you can remove the version constraint for the tests to pass.

Reply:
Thanks.